docs/package.json

```diff
@@ -13,24 +13,28 @@
   },
   "dependencies": {
     "@algolia/client-search": "^4.13.1",
-    "@astrojs/preact": "^1.0.2",
-    "@astrojs/react": "^1.1.1",
-    "@docsearch/css": "^3.1.0",
-    "@docsearch/react": "^3.1.0",
-    "@types/react": "^17.0.45",
-    "astro": "1.3.0",
-    "hastscript": "^7.0.2",
+    "@astrojs/preact": "^1.2.0",
+    "@astrojs/react": "^1.2.1",
+    "@astrojs/tailwind": "^2.1.1",
+    "@docsearch/css": "^3.2.2",
+    "@docsearch/react": "^3.2.2",
+    "@types/react": "^18.0.22",
+    "astro": "1.5.2",
+    "clsx": "^1.2.1",
+    "hastscript": "^7.1.0",
     "html-escaper": "^3.0.3",
-    "media-chrome": "^0.11.1",
-    "preact": "^10.7.3",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
+    "media-chrome": "^0.14.1",
```

Comment:
non-blocking: we should figure out how to keep this up-to-date. Maybe we can use yarn's `file:` here to point at the root of this project?
Alternatively, we can do a script as part of a publish that updates this version on publish.

Reply:
If we can use `file:` then having the individual component examples in the docs site won't be a huge issue because then if we wanted to debug an individual example it would just be all linked together.

Reply:
good call, will make this change